bot/core/lua/lua_py_dict.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import annotations
+
+from typing import Any
+
+MAX_LUA_INT = 982688721935675392
+
+
+class LuaPyDict(dict):  # type: ignore
+    def __getitem__(self, key: str) -> Any:
+        val = super().__getitem__(key)
+        if isinstance(val, dict):
+            return LuaPyDict(val)
+        if isinstance(val, int) and val > MAX_LUA_INT:
+            return str(val)
```

Comment:
the value won't be a hikari.Snowflake because it was converted to `python.int` after being received from `lua.number`. Although, yes, it is *because* they are discord ids, but this check won't work.

Reply:
ok